main.go

```diff
@@ -148,6 +149,12 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "TransportURL")
 		os.Exit(1)
 	}
+	if strings.ToLower(os.Getenv("ENABLE_WEBHOOKS")) != "false" {
```

Comment:
https://pkg.go.dev/strconv#ParseBool would be a more user friendly approach to allow multiple bool like string values

Reply:
@gibizer that may be but as we already have prior art for this code in our operators we should probably call stuff like this out in a separate issue right?

Reply:
I can file a separate issue for it as it is definitely not a bug just an improvement. (However I start to feel that we never have the time in the future to go and work on those improvement stories.)